helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -272,6 +277,7 @@ public Job(
     this.secondsToWaitBeforeKill = secondsToWaitBeforeKill;
     this.ramdisks = firstNonNull(ramdisks, EMPTY_RAMDISKS);
     this.rolloutOptions = rolloutOptions;
+    this.runtime = Optional.fromNullable(runtime).orNull();
```

Comment:
Appears that you can do `this.runtime = runtime` here.

Reply:
Hmm, you're right. I wonder why other fields follow this pattern.

Reply:
It is a weird pattern that I don't think should be continued. This code used to do a lot of `Optional.fromNullable(foo).orElse(blah).get()`, but at some point that was changed to use `MoreObjects.firstNonNull(a, b)` as there is no real reason to create an `Optional` instance that is then discarded immediately.

Reply:
Will address this in a separate PR.